examples/docs-examples/tutorials/accessing-internet/outbound-ipfs.mjs

```diff
@@ -0,0 +1,33 @@
+import { TaskExecutor } from "@golem-sdk/golem-js";
+import { readFile } from "fs/promises";
+
+const url = "https://ipfs.io/ipfs/bafybeihkoviema7g3gxyt6la7vd5ho32ictqbilu3wnlo3rs7ewhnp7lly";
+
+(async function main() {
+  // Load the manifest.
+  const manifest = await readFile(`./manifest.json`);
```

Comment:
If we wanted to test it automatically, the `manifest.json` file is missing in this directory. Additionally, you need to add this file to the list of automatic tests in`tests/examples/examples.json`